include/swift/Basic/Sanitizers.h

```diff
@@ -21,6 +21,26 @@ enum class SanitizerKind : unsigned {
   Thread,
 };
 
+// This is mostly the same as LLVM's llvm::SanitizerCoverageOptions.
+// It exists purely to avoid including "llvm/Transforms/Instrumentation.h"
+// in Swift header files.
+struct SanitizerCoverageOpts {
+  enum Type : unsigned {
+    None = 0, ///< No coverage instrumentation
+    Function, ///< Record coverage for function entry
+    BB,       ///< Record coverage for basic block entry
+    Edge,     ///< Record coverage for basic block edges
+  } CoverageType : 2;
+  // Experimental features
+  bool IndirectCalls : 1; ///< Record coverage for indirect caller-callee pairs
```

Comment:
This might be cargo-culting, but IIRC MSVC has odd behavior with boolean bitfields, and we've avoided them elsewhere in the codebase (using `unsigned` instead).